app/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/SubsidyController.scala

```diff
@@ -139,6 +151,107 @@ class SubsidyController @Inject() (
       }
   }
 
+  def getReportPaymentFirstTimeUser: Action[AnyContent] = verifiedEmail.async { implicit request =>
+    withLeadUndertaking { _ =>
+      renderFormIfEligible { journey =>
+        val updatedForm =
+          journey.reportPaymentFirstTimeUser.value
+            .fold(reportedPaymentFirstTimeUserForm)(v => reportedPaymentFirstTimeUserForm.fill(FormValues(v)))
+        Ok(reportPaymentFirstTimeUserPage(updatedForm, routes.AccountController.getAccountPage.url))
+      }
+    }
+  }
+
+  def postReportPaymentFirstTimeUser: Action[AnyContent] = verifiedEmail.async { implicit request =>
+    withLeadUndertaking { _ =>
+      implicit val eori: EORI = request.eoriNumber
+
+      def handleValidFormSubmission(f: FormValues): OptionT[Future, Result] = {
```

Comment:
shift-f6 rename this to formValues. 
It will autosuggest
no need to type less and f has no meaning

Reply:
understood  can we resolve later plz